automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingApiSpec.scala

```diff
@@ -0,0 +1,129 @@
+        }
+
+        // add studentA to google billing project with USER role
+        val studentA = UserPool.chooseStudent
+        Rawls.billing.addUserToBillingProject(billingProjectName, owner.email, BillingProjectRole.Owner)
+        Rawls.billing.addUserToBillingProject(billingProjectName, studentA.email, BillingProjectRole.User)
+
+        // verify studentA is a member in google billing project
+        val members: List[Map[String, String]] = Rawls.billing.listMembersInBillingProject(billingProjectName)
+        val emails: List[String] = members.map(_.getOrElse("email", ""))
+        emails should (contain(studentA.email) and not contain "")
+
+        // create new workspace
+        val workspaceName = randomIdWithPrefix("BillingApiSpec")
+        register cleanUp Try(Rawls.workspaces.delete(billingProjectName, workspaceName)).recover {
+          case _: RestException =>
+        }
+        Rawls.workspaces.create(billingProjectName, workspaceName)
+        Orchestration.workspaces.waitForBucketReadAccess(billingProjectName, workspaceName)
+
+        // create a method
+        withMethod("BillingApiSpec_workspace", MethodData.SimpleMethod) { methodName =>
+          val method: Method = MethodData.SimpleMethod.copy(methodName = methodName)
+
+         // val configNamespace = billingProjectName
+         // val configName = s"${workspaceName}_${SimpleMethodConfig.configName}"
+          val participantId = randomIdWithPrefix("participant")
+          val participantEntity = s"entity:participant_id\n$participantId"
+          Orchestration.importMetaData(billingProjectName, workspaceName, "entities", participantEntity)
+
+          // create method config in workspace
+          Rawls.methodConfigs.createMethodConfigInWorkspace(
+            billingProjectName,
+            workspaceName,
+            method,
+            SimpleMethodConfig.configNamespace,
+            SimpleMethodConfig.configName,
+            1,
+            SimpleMethodConfig.inputs,
+            SimpleMethodConfig.outputs,
+            SimpleMethodConfig.rootEntityType)
+
+          // launch submission
+          val submissionId = Rawls.submissions.launchWorkflow(
+            billingProjectName,
+            workspaceName,
+            SimpleMethodConfig.configNamespace,
+            SimpleMethodConfig.configName,
+            SimpleMethodConfig.rootEntityType,
+            participantId,
+            "this",
+            useCallCache = false)
+
+          // pause a minute because cromwell isn't fast
+          Thread.sleep(60 * 1000)
+
+          // monitor submission status until Done
+          val submissionStatus = "Done" // submission complete successfully
+          Submission.waitUntilSubmissionIsStatus(billingProjectName, workspaceName, submissionId, submissionStatus)
+        }
+
+        // clean up
+        Rawls.workspaces.delete(billingProjectName, workspaceName)
```

Comment:
Will the registered `cleanUp`s on lines 45 and 68 take care of this?

Reply:
explicitly calling the delete here is because I want to see the test fails if delete fails.